File: c/bee1048.c
#include <stdio.h>

int main() {
    double salario, percent, novoSalario, reaj;
    scanf("%lf", &salario);

    if(salario<=400)
        percent = 0.15;
    else if(salario > 400 && salario <= 800)
        percent = 0.12;
    else if(salario > 800 && salario <= 1200)
        percent = 0.1;
    else if(salario > 1200 && salario <= 2000)
        percent = 0.07;
    else if(salario > 2000)
        percent = 0.04;

    reaj = salario*percent;
    novoSalario = salario + reaj;

    printf("Novo salario: %.2lf\n", novoSalario);
    printf("Reajuste ganho: %.2lf\n", reaj);
    printf("Em percentual: %.0lf %%\n", percent*100);
    
    return 0;
}

File: beecrowd/python/bee1048.py
sal = float(input())

if 0 <= sal <= 400:
    num_r = 0.15
    new_sal = sal + (sal*num_r)
    r = sal * num_r
elif 400 < sal <= 800:
    num_r = 0.12
    new_sal = sal + (sal*num_r)
    r = sal * num_r
elif 800 < sal <= 1200:
    num_r = 0.1
    new_sal = sal + (sal*num_r)
    r = sal * num_r
elif 1200 < sal <= 2000:
    num_r = 0.07
    new_sal = sal + (sal*num_r)
    r = sal * num_r
else:
    num_r = 0.04
    new_sal = sal + (sal*num_r)
    r = sal * num_r

print(f'Novo salario: {new_sal:.2f}')
print(f'Reajuste ganho: {r:.2f}')
print(f'Em percentual: {num_r*100:.0f} %')
